hera_cal/abscal.py

```diff
@@ -532,9 +602,63 @@ def ant_dly_phi_gain_arr(self):
         else:
             return None
 
+    @property
+    def dly_slope(self):
+        """ return _dly_slope dict """
+        if hasattr(self, '_dly_slope'):
+            return copy.deepcopy(self._dly_slope)
+        else:
+            return None
+
+    @property
+    def dly_slope_gain(self):
+        """ form complex gain from _dly_slope dict """
+        if hasattr(self, '_dly_slope'):
+            dly_slope = self.dly_slope
+            return odict(map(lambda k: (k, np.exp(2j*np.pi*self.freqs.reshape(1, -1)*np.einsum("i...,i->...", dly_slope[k], self.antpos[k[0]][:2]))), flatten(self._gain_keys)))
+        else:
+            return None
+
+    def custom_dly_slope_gain(self, gain_keys, antpos):
```

Comment:
This is line is too long. Also, can you add some comments to explain what's going on here...it's really hard to read.